assets/scripts/segments/Building.jsx

```diff
@@ -122,13 +121,10 @@ class Building extends React.Component {
   handleKeyDown = (event) => {
     if (!this.state.isEditable) return
 
-    const negative = (event.keyCode === KEYS.MINUS) ||
-      (event.keyCode === KEYS.MINUS_ALT) ||
-      (event.keyCode === KEYS.MINUS_KEYPAD)
+    const negative = (event.key === '-')
 
-    const positive = (event.keyCode === KEYS.EQUAL) ||
-      (event.keyCode === KEYS.EQUAL_ALT) ||
-      (event.keyCode === KEYS.PLUS_KEYPAD)
+    const positive = (event.key === '=') ||
```

Comment:
Previously we had to check for the "=" key because it shared the same position as the "+" character (at least on US English keyboards). With the new `.key` property detecting the actual input character, we should be able to remove this check for `=`.

Reply:
Hey, thanks for the comment! Removed the check for '=' in my newest commit. Thank you for this opportunity to contribute and I understand if I PR became redundant due to other people's work. I still appreciate it.